Nexpo/Controllers/CompaniesController.cs

```diff
@@ -165,6 +165,32 @@ public async Task<ActionResult> PutMe(UpdateCompanySelfDto dto)
             }
             await _companyRepo.Update(company);
 
+            return Ok(company);
+        }
+
+         /// <summary>
+        /// Add new company
+        /// </summary>
+        [HttpPut]
```

Comment:
Should be [HttpPost] instead of [HttpPut] as you are adding a new company and not changing an already existing one. This change is solely conventional and should not require any other changes than just changing this line.